poller/poller.go

```diff
@@ -64,8 +67,9 @@ func NewPoller(splitioAPIKey string, pollingRateSeconds int, serializeSegments b
 		splitio = api.NewSplitioAPIBinding(splitioAPIKey, "")
 	}
 	emptyCache := Cache{
-		SplitData:      SplitData{},
-		SerializedData: emptyCacheLoggingScript,
+		splitData:             SplitData{},
+		serializedData:        emptyCacheLoggingScript,
+		serializedDataSubsets: make(map[string]string),
```

Comment:
I think it will help for easier understanding if we document here (or somewhere else) explains a little bit of this map. (e.g the key will be subset of sorted split names combined with a `'.'` e.g `"split_1.split_2"`)